vendor/bat-native-ads/src/bat/ads/internal/conversions/conversions.cc

```diff
@@ -98,19 +83,25 @@ void Conversions::MaybeConvert(const std::vector<std::string>& redirect_chain) {
 }
 
 void Conversions::StartTimerIfReady() {
-  DCHECK(is_initialized_);
+  database::table::ConversionQueue database_table;
+  database_table.GetAll(
+      [=](const Result result,
```

Comment:
`=` capture not needed

Reply:
ignore since we do need capture because member functions are called inside lambda